### ISSUE TITLE
[Bug] Camera' object has no attribute 'motion'

### ISSUE THREAD

**Issue by paulcusick3** [SecKatie/ha-wyzeapi#523]
**Describe the bug**
Do not see issue, but filling up logs, reported over 3k times in 24hours.  Guessing it is either door bell or floodlight pro that I recently installed.

**To Reproduce**
N/A

**Expected behavior**
N/A

**System configuration**
System: Home Assistant Blue / Hardkernel ODROID-N2/N2+
HA Version: Home Assistant 2023.10.1
WyzeApi Version: 

**home-assistant.log**

```
2023-10-12 10:49:22.796 ERROR (MainThread) [homeassistant.util.logging] Exception in handle_camera_update when dispatching 'wyzeapi.camera_updated-LD_CFP_D03F276D3948': (<Device: DeviceTypes.CAMERA, LD_CFP_D03F276D3948>,)
Traceback (most recent call last):
  File "/config/custom_components/wyzeapi/switch.py", line 455, in handle_camera_update
    self.async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 779, in async_write_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 879, in _async_write_ha_state
    state, attr = self._async_generate_attributes()
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 820, in _async_generate_attributes
    state = self._stringify_state(available)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 785, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1307, in state
    if (is_on := self.is_on) is None:
                 ^^^^^^^^^^
  File "/config/custom_components/wyzeapi/switch.py", line 444, in is_on
    return self._device.motion
           ^^^^^^^^^^^^^^^^^^^
AttributeError: 'Camera' object has no attribute 'motion'
```

**Comment by paulcusick3** [comment]
Flood light pro = LD_CFP
Doorbell Pro = GW_BE1
Lock Bolt = unknown - just pulling battery status as unknown

**Comment by brg468** [comment]
Oh yea I just noticed the doorbell is already excluded. I’ll submit a PR but there’s been no activity on this repo in a while so I wouldn’t hold your breath on getting it merged anytime soon. You can add the model number here and that should exclude it. 

https://github.com/SecKatie/ha-wyzeapi/blob/5ccebafe3266ceb26bceadfd73f0ce4e2124477e/custom_components/wyzeapi/switch.py#L27

**Comment by paulcusick3** [comment]
> Oh yea I just noticed the doorbell is already excluded. I’ll submit a PR but there’s been no activity on this repo in a while so I wouldn’t hold your breath on getting it merged anytime soon. You can add the model number here and that should exclude it.
> 
> https://github.com/SecKatie/ha-wyzeapi/blob/5ccebafe3266ceb26bceadfd73f0ce4e2124477e/custom_components/wyzeapi/switch.py#L27

Thanks - Updated locally to exclude

**Comment by SecKatie** [comment]
Fixed in #524
